en/06_Texture_mapping/02_Combined_image_sampler.adoc

```diff
@@ -5,7 +5,7 @@
 == Introduction
 
 We looked at descriptors for the first time in the uniform buffers part of the tutorial.
-In this chapter we will look at a new type of descriptor: _combined image sampler_.
+In this chapter, we will look at a new type of descriptor: _combined image sampler_.
```

Comment:
We might want to rephrase that a bit. This sounds like you MUST use a combined image sampler to sample an image in a shader, which isn't the case.